### DISCUSSION TITLE
Why i can not get connected with defualt host and port?

### DISCUSSION THREAD

**Discussion by PengyuanHAN** [milvus-io/milvus#6293] [Q&A and General discussion]
Why i can not get connected with defualt host and port?

host = '127.0.0.1'
port = '19530'
connections.connect("default", host=host, port=port)

timeout with these codes.

**Comment by xiaofan-luan** [comment]
did you tried to telnet and see if network is reachable?

**Reply by PengyuanHAN** [reply]
What do you mean?
I mean, what is the server.

**Reply by xiaofan-luan** [reply]
try run the command: telnet localhost 19530, see if it looks like as follow.
![image](https://user-images.githubusercontent.com/83447078/124374542-b3a15300-dcce-11eb-82b3-4bdefb7eba6a.png)
also what it behavior like? connect blocked a while and terminated or it stucks?

**Reply by PengyuanHAN** [reply]
Thank you. I think i know what happend.
